soccer/gameplay/robocup-py.cpp

```diff
@@ -459,6 +459,11 @@ void WinEval_add_excluded_robot(WindowEvaluator* self, Robot* robot) {
 
 BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(Point_overloads, normalized, 0, 1)
 
+float Point_get_x(const Geometry2d::Point* self) { return self->x(); }
+float Point_get_y(const Geometry2d::Point* self) { return self->y(); }
```

Comment:
Yeah I didn't check. Yeah the setter won't work since you don't have a setter in Point.